src/content/app/genome-browser/components/browser-track-legend/BrowserTrackLegend.scss

```diff
@@ -0,0 +1,33 @@
+.anchor {
+  position: absolute;
+}
+
+.actualAnchor {
+  position: relative;
+  width: 0;
+  height: 0;
+}
```

Comment:
Is this styling used anywhere?

Reply:
Ah, good point. That was me struggling with the tooltip for the left corner of a track. Deleted now.